src/main/smithy/HttpErrors.smithy

```diff
@@ -7,8 +7,6 @@ use smithy4s.meta#vector
 @error("client")
 @httpError(400)
 structure BadRequest {
-    @required
-    code: Integer = 400
```

Comment:
This is an anti-pattern: it would lead to the status code being sent in the http-body, and allows for the developer to change the value in a way that doesn't guarantee consistency with the actual http status code that will be sent.